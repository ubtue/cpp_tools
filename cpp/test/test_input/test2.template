And the list is:
{LOOP item,price}{item} costs ${price}.
{ENDLOOP}DONE.
