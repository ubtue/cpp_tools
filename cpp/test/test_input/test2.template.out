And the list is:
fish costs $3.55.
cheese costs $2.99.
DONE.
